blocks/init/src/Blocks/custom/lottie/components/lottie-settings.js

```diff
@@ -0,0 +1,99 @@
+import { __ } from '@wordpress/i18n';
+import { Responsive, checkAttrResponsive, getAttrKey, icons, ucfirst, IconLabel} from '@eightshift/frontend-libs/scripts';
+import { BaseControl, Button, Icon, ButtonGroup } from '@wordpress/components';
+import globalManifest from '../../../manifest.json';
+
+export const LottieSettings = ({
+	attributes,
+	setAttributes,
+	manifest,
+}) => {
+	const {
+		attributes: manifestAttributes,
+		responsiveAttributes: {
+			lottieLoop
+		},
+	} = manifest;
+
+	const {
+		globalVariables: {
+			breakpoints,
+		},
+	} = globalManifest;
+
+	const lottieTriggerValue = checkAttrResponsive('lottieTrigger', attributes, manifest, true);
+	const lottieUrlValue = checkAttrResponsive('lottieUrl', attributes, manifest);
```

Comment:
I like to keep variable names the same as attribute names when ever that is possible so please check if this is possible